<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1566902476506' stop='1566902478699'>
    <name>it should check multiform</name>
    <title>it should check multiform</title>
    <test-cases>
        <test-case start='1566902476507' status='failed' stop='1566902478699'>
            <name>should check the title</name>
            <title>should check the title</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='7554f696-ec12-481a-b95e-00b645e19b7f-attachment.png' type='image/png' size='20101'/>
            </attachments>
            <failure>
                <message>Failed: Error while waiting for Protractor to sync with the page: "both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor's bootstrapping.  See http://git.io/v4gXM for details"</message>
                <stack-trace>Error: Error while waiting for Protractor to sync with the page: "both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor's bootstrapping.  See http://git.io/v4gXM for details"
                    at /usr/local/lib/node_modules/protractor/built/browser.js:463:23
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:85:5)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as getText] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.&lt;computed> [as getText] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at UserContext.&lt;anonymous> (/Users/prakash/Desktop/automation/practice/spec.js:36:21)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
                    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
                    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
                    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                From: Task: Run it("should check the title") in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (/Users/prakash/Desktop/automation/practice/spec.js:34:5)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/Users/prakash/Desktop/automation/practice/spec.js:28:1)
                    at Module._compile (internal/modules/cjs/loader.js:776:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
                    at Module.load (internal/modules/cjs/loader.js:643:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:556:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>